orbstation/modules/table_shuffle/table_shuffle_subsystem.dm

```diff
@@ -314,39 +314,69 @@ SUBSYSTEM_DEF(table_shuffle)
 	var/vends = 0
 	var/moves = 0
 	var/decays = 0
+	var/non_events = 0
 	for(var/obj/item/I in all_moves)
 		var/stats = describe_journey(A,I,all_moves[I],table_turfs,destinations)
+		if(!stats)
+			non_events++
+			continue
 		if(stats & 1) vends++
 		if(stats & 2) moves++
 		if(stats & 4) decays++
-	A.shuffle_log += "<p><b>In total, [vends] free items and [decays] consumed items out of [all_moves.len] events.</b></p>"
+	. = all_moves.len - non_events
+	if(. > 0)
+		A.shuffle_log += "<p><b>In total, [vends] free items and [decays] consumed items out of [all_moves.len - non_events] events.</b></p>"
 	if(!manual)
 		total_vends += vends
 		total_moves += moves
 		total_decays += decays
-
-	return all_moves.len
+		if(. == 0)
+			affected_area_amt-- // we included it but nothing happened, don't contribute to total
 
 /datum/controller/subsystem/table_shuffle/proc/shuffle_decay_item(var/opt,var/area/A, var/obj/item/I, var/list/source_turfs,var/list/candidate_turfs, var/list/candidate_closets,var/list/item_log,var/deferred=0)
-	if(opt & SHUFFLE_DECAY)
-		if(istype(I,/obj/item/food))
-			if(ispath(I:trash_type) && prob(prob_eat))
-				var/J = new I:trash_type(I.loc)
-				item_log[1]=J
-				item_log[4]=200 // item has transformed, will be qdel'd in describe_journey
-				I = J
+	var/obj/effect/decal/cleanable/extra_dirt = null
+	if((opt & SHUFFLE_DECAY) && prob(prob_eat))
+		if(istype(I,/obj/item/food) && ispath(I:trash_type))
+			var/J = new I:trash_type(I.loc)
+			item_log[1]=J
+			item_log[4]=200 // item has transformed, will be qdel'd in describe_journey
```

Comment:
This isn't new code but I missed it last time, what does `item_log[4] = 200` (or for that matter `item_log[1]` and `item_log[4] = 100`) mean?

Reply:
200 is a magic number, and I suppose I ought to add constants that explain what the item log indexes mean.  Slot four is "Percent consumption", so 100% just means it was completely consumed.  200% means not only was it completely consumed, but it transformed into another object.